packages/bull-vault/src/BullStrategy.sol

```diff
@@ -72,6 +73,22 @@ contract BullStrategy is ERC20, LeverageBull {
         require(msg.sender == weth || msg.sender == address(crab), "BS0");
     }
 
+    /**
+     * @notice withdraw airdropped asset
+     * @dev can only be called by owner
+     * @param _asset asset address
+     * @param _receiver receiver address
+     */
+    function farm(address _asset, address _receiver) external onlyOwner {
```

Comment:
are euler tokens just airdropped to the strategy? or do we have to claim them?

Reply:
https://github.com/euler-xyz/euler-contracts/blob/master/contracts/mining/EulDistributor.sol